src/connectionController.ts

```diff
@@ -170,6 +170,15 @@ export default class ConnectionController {
         []
       );
 
+    if (connectionIdsThatDidNotMigrateEarlier.length > 0) {
+      try {
+        ext.keytarModule =
+          ext.keytarModule === undefined ? createKeytar() : ext.keytarModule;
```

Comment:
I know this is pre-existing code, but this could just be

```suggestion
        ext.keytarModule ??= createKeytar();
```

right?

Reply:
Hmmm, it doesn't have the same behaviour (sadly). `??=` will also overwrite keytarModule on null, and we only want to do it on undefined (due existing test behaviour).